bundles/org.eclipse.cdt.lsp.clangd/src/org/eclipse/cdt/lsp/internal/clangd/ClangdFallbackManager.java

```diff
@@ -19,8 +19,8 @@
 import org.eclipse.cdt.core.build.ICBuildConfiguration;
 import org.eclipse.cdt.core.build.ICBuildConfigurationManager;
 import org.eclipse.cdt.core.parser.IScannerInfo;
-import org.eclipse.cdt.lsp.InitialUri;
 import org.eclipse.cdt.lsp.ExistingResource;
+import org.eclipse.cdt.lsp.InitialUri;
```

Comment:
Do we use it?

Reply:
Currently yes. But it seems to me, that they made some improvemetns in clangd 16.0.2 regarding the `System includes extractor`:
```txt
I[07:23:28.886] System includes extractor: successfully executed C:\msys2\msys64\mingw64\bin\clang.exe
	got includes: "C:/msys2/msys64/mingw64/include/c++/12.2.0, C:/msys2/msys64/mingw64/include/c++/12.2.0/x86_64-w64-mingw32, C:/msys2/msys64/mingw64/include/c++/12.2.0/backward, C:/msys2/msys64/mingw64/lib/clang/15.0.7/include, C:/msys2/msys64/mingw64/include"
	got target: "x86_64-w64-windows-gnu"
I[07:23:28.890] --> textDocument/publishDiagnostics
I[07:23:28.892] Failed to find compilation database for C:\msys2\msys64\mingw64\include\c++\12.2.0\string
I[07:23:28.892] ASTWorker building file C:\msys2\msys64\mingw64\include\c++\12.2.0\string version 1 with command clangd fallback
```
It seems that the `System includes extractor` would solve our problem here. But I want to verify it a little more.